typescript/api-report/purchases-typescript-internal.api.md

```diff
@@ -298,7 +310,10 @@ export interface PurchasesConfiguration {
     apiKey: string;
     appUserID?: string | null;
     entitlementVerificationMode?: ENTITLEMENT_VERIFICATION_MODE;
+    // @deprecated
     observerMode?: boolean;
+    pendingTransactionsForPrepaidPlansEnabled?: boolean;

```

Comment:
Note that I'm keeping the `observerMode` property to avoid the breaking change, but `purchasesAreCompletedBy` has priority if it's set over `observerMode`.

Reply:
Imo a breaking change in an internal library is okay, to make sure we always use the correct api call. Or is PHC not strictly internal?

Reply:
Well this is a bit confusing actually, this typescript library actually provides some interfaces that are used as the public interface in the React Native and Capacitor SDKs (we basically expose these interfaces from those libraries). So a breaking change here would mean a breaking change in those 2 SDKs and I would like to avoid that. 

If it was only PHC I would completely agree with you, though I would probably make a major just in case (there have been people using PHC at some point 😅 )

Reply:
Oh yes that totally makes sense then! Thanks for the context!